src/relayserver/ServerConfigParams.ts

```diff
@@ -30,11 +31,12 @@ export interface ServerConfigParams {
   readyTimeout: number
   devMode: boolean
   registrationBlockRate: number
-  maxAcceptanceBudget: number
+  maxRelayExposure: number
   alertedBlockDelay: number
   minAlertedDelayMS: number
   maxAlertedDelayMS: number
   trustedPaymasters: Address[]
+  blacklistedPaymasters: Address[]
```

Comment:
This is coming from an unapproved branch, doesn't it? Where is the rest of 'blacklist' commit here?

Reply:
Yes, it's part of another PR that this one is branched out from.